controllers/userAPI.js

```diff
@@ -1,74 +1,68 @@
 const express = require('express');
 
-const userModel = require('../models/user')
-const userView = require('../views/userView')
+const userModel = require('../models/user');
+const userView = require('../views/userView');
 const jsonResponse = require('../utils/JSONResponse');
 const auth = require('../utils/authUtil');
 
 const router = express.Router();
 
 // Every method is prepended with "/user" see app.js
 
-router.post('/register', (req, res) => 
-{
-	var user = req.body;
+router.post('/register', (req, res) => {
+    var user = req.body;
 
-	userModel.checkUserExists(user.username,
-		{
-			found: 
-				() => res.send(jsonResponse.fail("Could not register an already existing user")),
-			notFound: 
-				() => userModel.registerUser(req.body,
-					{
-						success: 
-							() => res.send(jsonResponse.success()),
-						fail: 
-							(reason) => res.send(jsonResponse.fail(reason))
-					})
-		});
+    userModel.checkUserExists(user.username,
```

Comment:
can you please fix your IDE so that the tabs arent replaced with spaces?

We want tabs :)